migration/mule/import-contacts-into-salesforce/README.md

```diff
@@ -0,0 +1,59 @@
+increase your familiarity with Integration Studio, consider completing one or more 
+[WSO2 EI Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/).
+
+### Example Use Case
+In this example, Contacts in a csv file is being imported to Salesforce using the File Connector and the Salesforce Connector. The response from Salesforce is then logged to track the status of creating each record.   
+
+### Set Up and Run the Example
+
+1. Start WSO2 Integration Studio ([Installing WSO2 Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/installing-WSO2-Integration-Studio/)).
+2. In your menu in Studio, click the **File** menu. In the File menu select the **Import...** item.
+3. In the Import window select the **Existing WSO2 Projects into workspace** under **WSO2** folder.
+4. Browse and select the file path to the downloaded sample of this github project 
+("import-contacts-into-salesforce" folder of the downloaded github repository).
+5. Lets add the file connector into the workspace. Right click on the **ImportContactsIntoSalesforce** and select 
+**Add or Remove Connector**. Keep the **Add connector** option selected and click **Next>**. Search for 'file' using the 
+search bar and click the download button located at the bottom right corner of the file connector. Click **Finish**.
+6. Similarly, add the Salesforce REST Connector to the workspace.
+7. Follow these [steps](https://ei.docs.wso2.com/en/latest/micro-integrator/references/connectors/salesforce-rest-connector/sf-access-token-generation/) to generate the Access Tokens for Salesforce and obtain the Access Token, and Refresh Token.
+8. Open the **Sequence.xml** under 
+**import-contacts-into-salesforce/ImportContactsIntoSalesforce/src/main/synapse-config/sequences/** directory. 
+Configure the following properties with the previously obtained values under **<salesforcerest.init>**.
+    - Access Token
+    - Refresh Token
+    - API URL (e.g.: https://<INSTANCE>.salesforce.com)
+9. Copy the **contacts.csv** file in **import-contacts-into-salesforce/ImportContactsIntoSalesforce/src/main/resources/** to a location on your computer. 
+Provide the path to this as the source under **<fileconnector.read>**
+E.g.:
+```xml
+<source>file:///Users/testuser/Desktop/contacts.csv</source>
+```
+10. Run the sample by right clicking on the **import-contacts-into-salesforce** project and selecting **Run as -> Run On Micro Integrator**.
+11. Following logs can be observed in the console log.
+```
+[2020-04-28 11:18:16,994]  INFO {org.apache.synapse.mediators.builtin.LogMediator} - To: http://www.w3.org/2005/08/addressing/anonymous, WSAction: , SOAPAction: , MessageID: urn:uuid:9f36fa4b-a851-45a1-945a-afdbd9f004ad, Direction: request, Payload: {"hasErrors":false,"results":[{"referenceId":"1","id":"0032v00003Un7V6AAJ"},{"referenceId":"2","id":"0032v00003Un7V7AAJ"}]}
+
+```
+
```

Comment:
We need to mention how to verify that the records have been populated at SF side. 
```
In your browser, access your Salesforce account, then navigate to the Contacts tab.
Use the drop-down menu to display All Contacts, then scan your contacts for two new entries:
John Doe
Jane Doe
```